modules/govuk_testing_tools/manifests/xvfb.pp

```diff
@@ -1,17 +1,20 @@
-# == Class: xvfb
+# == Class: govuk_testing_tools::xvfb
 #
 # Installs xvfb-server from the Ubuntu repos
-class xvfb {
+#
+class govuk_testing_tools::xvfb(
+  $ensure  = 'present',
+){
 
   package { 'xvfb':
-    ensure => 'present',
+    ensure => $ensure,
   }
 
   file { '/etc/init/xvfb.conf':
-    ensure => present,
-    owner  => root,
-    group  => root,
-    source => 'puppet:///modules/xvfb/xvfb.conf',
```

Comment:
Just for my own purposes: why does this no longer work?

Reply:
Make a note of it and when we move this to test-kitchen we can work through it and look at the failures.